src/main/java/org/jitsi/videobridge/cc/SimulcastController.java

```diff
@@ -1066,8 +1066,6 @@ void setOptimalIndex(int newOptimalIdx)
          */
         void rtcpTransform(ByteArrayBuffer baf)
         {
-            SeenFrame maxSentFrame = this.maxSentFrame;
-
```

Comment:
Like Boris said, the local variable here is intentional. What kind of problems do you see @bbaldino ?